code/__defines/jobs.dm

```diff
@@ -24,7 +24,7 @@
 // Factions
 // Used to know what what roles are allowed to play as which factions.
 // Note that independent isn't a faction by itself, as faction in this case means a megacorporation.
-#define SCC_ROLES list(COMMAND_ROLES, COMMAND_SUPPORT_ROLES, EQUIPMENT_ROLES, ALL_FACTION_ROLES)
+#define SCC_ROLES list(COMMAND_ROLES, COMMAND_SUPPORT_ROLES, EQUIPMENT_ROLES, /datum/job/visitor) // Does not use ALL_FACTION_ROLES by lore team decision
```

Comment:
```suggestion
#define SCC_ROLES list(COMMAND_ROLES, COMMAND_SUPPORT_ROLES, EQUIPMENT_ROLES, /datum/job/visitor)
```

Reply:
This is to prevent an accidental re-addition as previously done in the mentioned PR.

Reply:
its pretty self-explanatory without the comment.